includes/tracks/events/class-wc-coupons-tracking.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * WooCommerce Coupons Tracking
+ *
+ * @package WooCommerce\Tracks
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * This class adds actions to track usage of WooCommerce Orders.
+ */
+class WC_Coupons_Tracking {
+	/**
+	 * Init tracking.
+	 */
+	public function init() {
+		add_action( 'load-edit.php', array( $this, 'tracks_coupons_events' ), 10 );
+	}
+
+	/**
+	 * Add a listener on the "Apply" button to track bulk actions.
+	 */
+	public function tracks_coupons_bulk_actions() {
+		wc_enqueue_js(
+			"
+			function onApplyBulkActions() {
+				var action = $('select[name=\"action\"]').val();
```

Comment:
Thanks for the fixes, @psealock. I just noticed something else that I missed in the first review. `$('select[name=\"action\"]').val()` will only work to get the selected value for the select box on the top of the coupons table. The name of the select box on the bottom of the table is `action2` and because of that, the tracker event is not fired when this select box is used to define the action to perform.